jira/client.py

```diff
@@ -771,6 +784,9 @@ def _fetch_pages(
                     and (total is None or page_start < total)
                     and len(next_items_page) == page_size
                 ):
+                    page_params = (
```

Comment:
If omitted the mock calls will point to the same page-params object, which causes retroactive changes to already fired mock calls